draft-ietf-scitt-architecture.md

```diff
@@ -397,9 +397,7 @@ Multi-tenant support can be enabled through the use of identifiers in the `iss`
 
 Registration Policies refer to additional checks over and above the Mandatory Registration Checks that are performed before a Signed Statement is accepted to be registered to the Append-only Log.
 
-Transparency Services MUST maintain Registration Policies.
-
-Transparency Services MUST also maintain a list of trust anchors, which SHOULD be used by Relying Parties to authenticate Issuers, and which MAY be included in a Registration Policy statement.
+Transparency Services MUST maintain Registration Policies and a list of trust anchors to authenticate Issuers upon Registration.
```

Comment:
Transparency Services MUST maintain Registration Policies.

Transparency Services MUST maintain a list of trust anchors as described in _TBD_.

Transparency Services MUST authenticate signed statements as part of a Registration Policy.